ere-guests/stateless-validator/reth/openvm/src/crypto.rs

```diff
@@ -0,0 +1,281 @@
+//! OpenVM Crypto Implementation for REVM
```

Comment:
This is copied from https://github.com/axiom-crypto/openvm-reth-benchmark/pull/523/files#diff-b60672a7a7c82dd3fe1bf292920be95d0f6d351e70f568d091c88d6820b86f71.

For now decided to do this for simplicity. After we have a version correctly running, we can try using the crate directly and see if many other reth dependencies don't interfere much.